src/lib/network/network_autoconfiguration.rb

```diff
@@ -43,10 +43,24 @@ def initialize
       Yast.include self, "network/routines.rb" # TODO: needed only for phy_connected
     end
 
-    # Checks if any of available interfaces is configured and active
+    # Checks if any of available interfaces is active and with a connection file present in the
+    # system
     #
     # returns [Boolean] true when at least one interface is active
     def any_iface_active?
+      Yast::Lan.Read(:cache)
+      config.interfaces.any? do |interface|
+        next false unless active_config?(interface.name)
+
+        config.connections.by_name(interface.name)
+      end
+    end
```

Comment:
I see quite some code duplication between this and `#network_configured?`

Reply:
Yep, I preferred to duplicate the logic by now, but could try to reorganize / extract some common logic.